compiler/src/dotty/tools/dotc/core/tasty/AttributePickler.scala

```diff
@@ -10,18 +10,15 @@ import java.nio.charset.StandardCharsets
 object AttributePickler:
 
   def pickleAttributes(
-    attributes: List[String],
+    scala2StandardLibrary: Boolean,
```

Comment:
I suggest you create a separate immutable `class Attributes` in this package that bundles all the possible attributes as fields. `pickleAttributes` would take an `Attributes` as argument and `AttributeUnpickler` would read and probably directly expose an instance of `Attributes`.

Reply:
Done